dt-metrics/counters/counter-post-stats.php

```diff
@@ -409,21 +456,68 @@ public static function get_connection_field_by_year( $post_type, $field, $connec
         $start = 0;
         $end = mktime( 24, 60, 60, 12, 31, $current_year );
 
-        $results = $wpdb->get_results(
+        // Determine total number of connections within time frame.
+        $connected = $wpdb->get_results(
             $wpdb->prepare( "
                 SELECT
-                    COUNT( DISTINCT log.object_id ) AS count,
+                    COUNT( DISTINCT p.ID, p.post_title ) AS count,
```

Comment:
@kodinkat, why add p.post_title in the distinct?

Reply:
@corsacca more so for completeness, but, agreed; we'll be able to get away with just the `p.ID`

Reply:
Lets simplify :)